tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -75,11 +75,21 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
         argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
         idStr = argTempBuffer;
         int tempId = std::stoi(idStr);
-        if (tempId >= 0 && (uint32_t)tempId < UINT32_MAX) {
-          rp.viewChangeTimeout = (uint32_t)tempId;
+        if (tempId >= 0 && (uint16_t)tempId < UINT16_MAX) {
+          rp.viewChangeTimeout = (uint16_t)tempId;
           rp.viewChangeEnabled = true;
         }
       } break;
+      case 'a': {
+        strncpy(argTempBuffer, optarg, sizeof(argTempBuffer) - 1);
+        argTempBuffer[sizeof(argTempBuffer) - 1] = 0;
+        idStr = argTempBuffer;
+        int tempId = std::stoi(idStr);
+        if (tempId >= 0 && (uint16_t)tempId < UINT16_MAX) {
```

Comment:
I know this is the same logic as above, but now that I think about it, we should assert on this rather than silently ignoring the parameter. In the future we should also allow much larger ints. We should just use size_t for everything parameter related instead of trying to optimize and have to do all this custom logic. If we want to restrict values we can do that specifically rather than relying on the size of the int.

Note that this doesn't have to be changed here, but is something we should likely track.

Reply:
I agree, but as you suggest, I prefer to tackle this separately.